public/video-ui/src/services/WorkflowApi.ts

```diff
@@ -278,7 +279,7 @@ export default class WorkflowApi {
     });
   }
 
-  static async updateNote({ id, note }: {id: string, note: string | null}): Promise<ApiResponse<number> | null> {
```

Comment:
What’s the rationale for this change? I _think_ I chose `null` over `undefined` because that was what I observer in the redux store, but are you confident that the type passed is actually `undefined`?

Reply:
Ah I see! Thanks for the explanation :)